bridges/centralized-ethereum/src/actors/dr_reporter.rs

```diff
@@ -84,8 +85,10 @@ impl Handler<DrReporterMsg> for DrReporter {
         };
 
         if msg.result.len() > self.max_result_size {
-            let radon_error = RadonErrors::BridgeOversizedResult as u8;
-            msg.result = vec![0xD8, 0x27, 0x81, 0x18, radon_error]
+            let radon_error =
+                RadError::try_from_kind_and_cbor_args(RadonErrors::BridgeOversizedResult, None)
+                    .unwrap();
+            msg.result = radon_error.encode_tagged_bytes().unwrap();
```

Comment:
I think we could use `?` syntax to replace some of the `unwrap`s if we modify the return type of the handler. Wdyt?